theme/theme.inc

```diff
@@ -25,7 +25,7 @@ function template_preprocess_islandora_internet_archive_bookreader(array &$varia
   }
   else {
     // Defaults to global viewer config.
-    $view_mode = (int) variable_get('islandora_internet_archive_bookreader_default_page_view', 1);
+    $view_mode = variable_get('islandora_internet_archive_bookreader_default_page_view', '1up');
```

Comment:
Same as my first comment. Not sure if we want to change all arguments on this side, maybe better to keep 1/2/3 etc? Other option would be to use the defaults in numeric form, but then cast/map them here to your wanted form?

Reply:
decided to update the condition that sets parms.mode https://github.com/Islandora/islandora_internet_archive_bookreader/pull/138/files#diff-542ab515026d028eec06ed5e93b11086R984-R994